test/CMakeLists.txt

```diff
@@ -80,4 +80,23 @@ set_test_target_parameters(benchmarks benchmarks.cc)
 add_subdirectory(system)
 if(CELERITY_DETAIL_INTEGRATION_TESTING)
   add_subdirectory(integration)
+endif()
+
+# Optionally link region map tests against Cairo for debug drawing
```

Comment:
This dependency is completely optional and just for internal debugging, but I wonder if we should still document also external dependencies somewhere at least.

Reply:
You think? This will only ever be relevant to someone working on the region map, which, realistically, is probably not going to be more than 2 people.

Reply:
That is also true. And I guess if you do work on it, you'll very quickly find out.